activities/codebase-stewardship/index.md

```diff
@@ -6,82 +6,84 @@ redirect_from:
 
 # Codebase stewardship
 
-We provide codebase stewardship at the codebase level.
-The [goals of codebase stewardship](goals.md) are to maximize the reuse of the [codebase](../../glossary/codebase-definition.md) and promote the long term sustainability of the community around the codebase.
-Through codebase stewardship, we help public source code and policy code products become successful.
+Codebase stewardship is what we call the way we interact with the public code codebase communities that aims to meet the Standard for Public Code.
+Our approach of being stewards is highly intentional.
```

Comment:
This sentence is a bit unclear to me

Reply:
Even in the context of the full paragraph? I was going for to make a statement, and then explaining it in the following sentences.